src/frontend/Sider/Behandling/Stønadsvilkår/PassBarn/PassBarn.tsx

```diff
@@ -25,7 +27,11 @@ const PassBarn: React.FC<Props> = ({ vilkårsregler, vilkårsvurdering }) => {
         (v) => v.vilkårType === Inngangsvilkårtype.PASS_BARN
     );
 
-    return vilkårsvurdering.grunnlag.barn.map((barn) => {
+    if (vilkårsvurdering.grunnlag['@type'] !== Stønadstype.BARNETILSYN) {
+        return <>Feil faktatype for pass av barn</>;
+    }
+    const grunnlagTilsynBarn = vilkårsvurdering.grunnlag as BehandlingFaktaTilsynBarn;
+    return grunnlagTilsynBarn.barn.map((barn) => {
```

Comment:
Trenger ingen typecasting her, kan beholde det som var
```suggestion
    return vilkårsvurdering.grunnlag.barn.map((barn) => {
```